_testutil_test.ts

```diff
@@ -1,43 +0,0 @@
-// deno-lint-ignore-file no-explicit-any
-import { assertType, type IsExact } from "@std/testing/types";
-
-import type { Equal, TypeChallengesEqual } from "./_testutil.ts";
-
-declare class Foo<T> {
-  declare private _: T;
-}
-
-Deno.test("IsExact gives false positive", () => {
-  // all of the following should be false
-  assertType<IsExact<() => void, (x?: number) => void>>(true);
-  assertType<IsExact<Foo<{ x: 1; a?: 1 }>, Foo<{ x: 1; b?: 1 }>>>(true);
-  assertType<IsExact<() => any, () => number>>(true);
-  assertType<IsExact<Foo<any>, Foo<number>>>(true);
-  assertType<IsExact<{ a: 1 }, { readonly a: 1 }>>(true);
-});
```

Comment:
This test now fails once we update @std/testing to 1.0.1 or higher, which is expected.